### PR TITLE
Normalize `az_tel` to the range of [0, 360) and compute sin_az_tel in dl1_to_dl2 step

### PR DESCRIPTION
**Pull request by SeiyaNozaki**
Related to #1079 

This PR normalize `az_tel` values to the range of [0, 360) degrees which is same as the distribution of MC data points. In addition, if there is no `sin_az_tel` in DL1 file because the DL1 was created using former version of lstchain, it computes `sin_az_tel` in dl1_to_dl2 step.

### CODE REVIEW

**Review by rlopezcoto — APPROVED** [review]

File: lstchain/reco/dl1_to_dl2.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 import numpy as np
 import pandas as pd
 from astropy.coordinates import SkyCoord
+from astropy.coordinates import Angle
```
Comment (chaimain) [inline lstchain/reco/dl1_to_dl2.py:17]: Combine these 2 imports

**Review by chaimain — APPROVED** [review]

**Comment by SeiyaNozaki** [comment]
Hi @vuillaut
Do you know which altitude value raises an error for the conversion? I think the range of altitude is within what astropy requires, no?

**Comment by maxnoe** [comment]
> and I sometimes get errors using the astropy converter on alt.

From simtel array files? Or from LST R0 files / pointing files?

I don't think we have ever observed at values that would trigger this errors.

The problem with simtelarray should be fixed, both in ctapipe:
https://github.com/cta-observatory/ctapipe/pull/2077

and astropy:
https://github.com/astropy/astropy/issues/13708

**Comment by vuillaut** [comment]
From what I can remember, that was on reconstructed alt, angles > 90 deg that I ended up filtering or clipping.
But I don't have a specific example at hand.

**Comment by SeiyaNozaki** [comment]
I have DL2 data with telescope zenith of 1 degree. I feel `reco_alt` was properly wrapped below 90 degrees. (maximum `reco_alt` was 89.9967188877843 degrees)
download (19)

**Pull request merged**